src/applications/ivc-champva/10-7959f-1/config/prefillTransformer.js

```diff
@@ -5,6 +5,7 @@ const prefillTransformer = (pages, formData, metadata) => {
       ...formData,
       veteranFullName: formData.veteranFullName,
       veteranAddress: formData.veteranAddress,
+      veteranResidentialAddress: formData.physicalAddress,
```

Comment:
Per Bryan's [new backend PR](https://github.com/department-of-veterans-affairs/vets-api/blob/4514125dba3cb4fd7c406eb27ad2c788299744d7/config/form_profile_mappings/10-7959F-1.yml), this line would need to be
`veteranResidentialAddress: formData.veteranPhysicalAddress,`